### DISCUSSION TITLE
To My Sponsors

### DISCUSSION THREAD

**Discussion by kolumnin** [community/community#28120] [Other Feature Feedback, Questions, & Ideas]
Please help me with a review and what is missing in my sponsor configurations.
Thank you.

**Comment by kolumnin** [comment]
I am from India. How to get a sponsored Dataline subscription for unfailing internet connectivity and speed daily for about 4 hours?

**Reply by jfullstackdev** [reply]
> sponsored Dataline subscription for unfailing internet connectivity and speed daily for about 4 hours?

that's not covered by GitHub Sponsors,

what you want to do is to actually create useful projects and someone or some company using them will sponsor you

**Reply by kolumnin** [reply]
Thank you @jdevstatic. That is useful.
2022-08-03-13:17@kolumnin
